subsys/bluetooth/host/cs.c

```diff
@@ -447,4 +447,197 @@ int bt_le_cs_remove_config(struct bt_conn *conn, uint8_t config_id)
 
 	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_REMOVE_CONFIG, buf, NULL);
 }
+
+int bt_le_cs_security_enable(struct bt_conn *conn)
+{
+	struct bt_hci_cp_le_security_enable *cp;
+	struct net_buff *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_CS_SECURITY_ENABLE, sizeof(*cp));
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	cp = net_buf_add(buf, sizeof(*cp));
+	cp->handle = sys_cpu_to_le16(conn->handle);
+
+	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_SECURITY_ENABLE, buf, NULL);
+}
+
+int bt_le_cs_procedure_enable(struct bt_conn *conn, const struct bt_le_cs_procedure_enable_param *params)
+{
+	struct bt_hci_cp_le_cs_procedure_enable *cp;
+	struct net_buff *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_CS_PROCEDURE_ENABLE, sizeof(*cp));
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	cp = net_buf_add(buf, sizeof(*cp));
+	cp->handle = sys_cpu_to_le16(conn->handle);
+	cp->config_id = params->config_id;
+	cp->enable = params->enable;
+
+	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_PROCEDURE_ENABLE, buf, NULL);
+}
+
+int bt_le_cs_set_procedure_parameters(struct bt_conn *conn, const struct bt_le_cs_set_procedure_parameters_param *params)
+{
+	struct bt_hci_cp_le_set_procedure_parameters *cp;
+	struct net_buff *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_CS_SET_PROCEDURE_PARAMETERS, sizeof(*cp));
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	cp = net_buf_add(buf, sizeof(*cp));
+	cp->handle = sys_cpu_to_le16(conn->handle);
+	cp->config_id = params->config_id;
+	cp->max_procedure_len = sys_cpu_to_le16(params->max_procedure_len);
+	cp->min_procedure_interval = sys_cpu_to_le16(params->min_procedure_interval);
+	cp->max_procedure_interval = sys_cpu_to_le16(params->max_procedure_interval);
+	cp->max_procedure_count = sys_cpu_to_le16(params->max_procedure_count);
+	sys_put_le24(params->min_subevent_len, cp->min_subevent_len);
+	sys_put_le24(params->max_subevent_len, cp->max_subevent_len);
+	cp->tone_antenna_config_selection = params->tone_antenna_config_selection;
+	cp->phy = params->phy;
+	cp->tx_power_delta = params->tx_power_delta;
+	cp->preferred_peer_antenna = params->preferred_peer_antenna;
+	cp->snr_control_initiator = params->snr_control_initiator;
+	cp->snr_control_reflector = params->snr_control_reflector;
+
+	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_SET_PROCEDURE_PARAMETERS, buf, NULL);
+}
+
+int bt_le_cs_set_channel_classification(uint8_t channel_classification[10])
+{
+	uint8_t *cp;
+	struct net_buff *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_CS_SET_CHANNEL_CLASSIFICATION, 10);
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	cp = net_buf_add(buf, 10);
+
+	if (sys_cpu_to_le16(0x0001) == 0x0001)
```

Comment:
I think you can just memcpy(). That's what we did here https://github.com/zephyrproject-rtos/zephyr/blob/4260f07a468aee09165d142a014c7759a55805d8/subsys/bluetooth/host/cs.c#L429

It does mean the input to this function needs to be HCI-formatted